synfig-studio/src/gui/app.cpp

```diff
@@ -1754,18 +1735,33 @@ App::App(const synfig::String& basepath, int *argc, char ***argv):
 		get_ui_interface()->error(_("Unknown exception caught when constructing App.\nThis software may be unstable."));
 	}
 
-	// Load sound effects
-	sound_render_done = new SoundProcessor();
-	sound_render_done->addSound(
-		SoundProcessor::PlayOptions(),
-		SoundProcessor::Sound(ResourceHelper::get_sound_path("renderdone.wav")));
+	// FIXME(rodolforg): Gtk::Application quits when all windows are hidden/closed
```

Comment:
Are you sure? As I see, Gtk::Application quits, when all windows **removed** from it.
https://developer-old.gnome.org/gtkmm/stable/classGtk_1_1Application.html#afe36cb32c2f4dc3ddbc35faae81cfb8a